### PR TITLE
Add LIPID MAPS fragment optimization dataset

### PR DESCRIPTION
**Pull request by ntBre**
## New Submission Checklist

- [x] Created a new folder in the submissions directory containing the dataset
- [x] Added `README.md` describing the dataset see [here](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2020-03-26-OpenFF-Gen-2-Torsion-Set-6-supplemental-2) for examples
- [x] All files used to produce the dataset are included with a description
- [x] Dataset follows the QCSubmit schema defined for [Datasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L340), [OptimizationDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1062) and [TorsionDriveDatasets](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L1225)
- [x] Dataset filename matches pattern `dataset*.json`; may feature a compression extension, such as `.bz2`
- [x] A PDF depicting the molecules is attached, in the case of torsiondrives this should include the highlighting of the central bond, this can be done automatically using [qcsubmit](https://github.com/openforcefield/qcsubmit/blob/56680a7d3298b5d8962edcb840b0fdb34558c053/qcsubmit/datasets.py#L854). 
- [x] QCSubmit validation passed
- [x] Made a new dataset entry in the mapping table in repository `README.md`
- [ ] Ready to submit!

### CODE REVIEW

File: submissions/2024-10-08-OpenFF-Lipid-Optimization-Training-Supplement-v1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+were also filtered out.
+
+## General Information
+
+* Date: 2024-10-08
+* Class: OpenFF Optimization Dataset
+* Purpose: Improve training coverage in Sage
+* Name: OpenFF Lipid Optimization Training Supplement v1.0
+* Number of unique molecules: 3971
+* Number of filtered molecules: 32
```
Comment (lilyminium) [inline submissions/2024-10-08-OpenFF-Lipid-Optimization-Training-Supplement-v1.0/README.md:27]: Just checking -- are these number of filtered molecules the ones that were filtered *out*, i.e. are not part of the number of unique molecules above?

**Review by lilyminium — APPROVED** [review]
Thanks Brent, this LGTM!

File: submissions/2024-10-08-OpenFF-Lipid-Optimization-Training-Supplement-v1.0/README.md (hunk 1 above)
Comment (ntBre) [reply submissions/2024-10-08-OpenFF-Lipid-Optimization-Training-Supplement-v1.0/README.md:27]: Yes that's right. This number is usually zero (at least in my previous submissions), but this time qcsubmit actually filtered out 32 molecules, I think for timeouts in conformer generation. My input SMILES file had 4003 entries, so these add up to that total at least.

Comment (lilyminium) [reply submissions/2024-10-08-OpenFF-Lipid-Optimization-Training-Supplement-v1.0/README.md:27]: Thanks Brent!

**Comment by j-wags** [comment]
We just got in trouble twice with the NRP admins for mis-resourced deployments so I'm scaling compute for this dataset up slowly. Expect throughput to increase 10-50x in the next few days if we don't hit any speedbumps.

**Pull request merged**